tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -697,6 +697,7 @@ public virtual void DoSomething()
 
             public void DoSomethingNonVirtual()
             {
+                this.WasCalled = true;

```

Comment:
Is this changed (and the one below) required as part of changing the specs to static?